### DISCUSSION TITLE
Always publish package with public access permission

### DISCUSSION THREAD

**Discussion by wcewlug** [community/community#30515] [Actions]
Whenever we publish a new package, it is private. Can we make public while pushing itself. I have tried modifying package permissions as Public but still every new package I publish has private permissions.

![Screenshot_2022-08-23_15_18_04](https://user-images.githubusercontent.com/8832856/186127874-47e0b369-d51d-464d-bbd9-c06dab4cb233.png)

**Comment by delosemre** [comment]
hello, have you checked here?
https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility

**Reply by naikrovek** [reply]
also, try publishing with a different name, instead of the same name over and over, and consider tagging the package with the version, instead of including it in the name.

`linuxdiary-3.0_1` might become `linuxdiary:3.0.1`

**Reply by naikrovek** [reply]
packages are a bit weird and are not just files being served.  some package ecosystems assume that once published, a package is _never_ going to change, so changes must be given a new version.  GitHub making this assumption (by keeping settings it has seen before for that package name) _may_ be a contributing factor here.
